stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/reactions/ReactionsAdapter.kt

```diff
@@ -0,0 +1,101 @@
+package io.getstream.chat.android.ui.messages.reactions
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.core.view.updateLayoutParams
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import io.getstream.chat.android.client.models.Reaction
+import io.getstream.chat.android.ui.R
+import io.getstream.chat.android.ui.databinding.StreamItemMessageReactionBinding
+import io.getstream.chat.android.ui.utils.UiUtils
+import io.getstream.chat.android.ui.utils.extensions.context
+import io.getstream.chat.android.ui.utils.extensions.getColorCompat
+
+public class ReactionsAdapter(
```

Comment:
Could this class be `internal`? Seems like we're not exposing it anywhere at the moment, its usage is `private` within `ReactionsView`.